com.arc.embeddedcdt/META-INF/MANIFEST.MF

```diff
@@ -23,14 +23,14 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.core.variables,
  org.eclipse.cdt.launch;bundle-version="7.1.0",
  org.eclipse.tm.terminal.control,
- org.eclipse.tm.terminal.connector.serial,
  org.eclipse.tm.terminal.view.ui,
  org.eclipse.tm.terminal.view.core,
  org.eclipse.cdt.dsf.gdb.ui,
  org.eclipse.cdt.dsf,
  org.eclipse.cdt.dsf.gdb,
- org.eclipse.cdt.launch.remote
+ org.eclipse.cdt.launch.remote,
+ org.eclipse.tm.terminal.connector.cdtserial
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Import-Package: com.ibm.icu.text,
- gnu.io
+ org.eclipse.cdt.serial
```

Comment:
I don't think this is needed because cdt.serial is a proper Eclipse plugin therefore should be imported via Require-Bundle. Please check if you can remove this line.

Reply:
It's needed (removing this line will result errors)